README.md

```diff
@@ -1,14 +1,14 @@
 # eslint-plugin-sonarjs [![npm version](https://badge.fury.io/js/eslint-plugin-sonarjs.svg)](https://badge.fury.io/js/eslint-plugin-sonarjs) [![Build Status](https://travis-ci.org/SonarSource/eslint-plugin-sonarjs.svg?branch=master)](https://travis-ci.org/SonarSource/eslint-plugin-sonarjs) [![Quality Gate](https://sonarcloud.io/api/project_badges/measure?project=eslint-plugin-sonarjs&metric=alert_status)](https://sonarcloud.io/dashboard?id=eslint-plugin-sonarjs) [![Coverage](https://sonarcloud.io/api/project_badges/measure?project=eslint-plugin-sonarjs&metric=coverage)](https://sonarcloud.io/dashboard?id=eslint-plugin-sonarjs)
 
-SonarJS rules for ESLint detecting bugs and suspicious patterns in your code.
+SonarJS rules for ESLint to detect bugs and suspicious patterns in your code.
```

Comment:
IMO this could go either way

Reply:
It sounds very awkward to me, but I can't place a specific grammar rule, so your call...

Reply:
I think it is because it is ambiguous, like "ESLint detecting bugs" sounds like one noun.